apache-solr/dashboards/apache-solr-logs-overview.libsonnet

```diff
@@ -0,0 +1,34 @@
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
```

Comment:
Looks like the logs aren't being parsed for severity levels correctly since everything is showing as grey. I would add a severity/level selector here and make sure its being parsed.
<img width="1679" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/4cd53b09-aa25-4d7e-ac11-04c64b79b079">

Reply:
Added in a levels selector. I do not however see levels like INFO showing up. Not sure what needs to be done as I looked for other examples and tried adding in extraFilters, but that just highlighted the level section in the logs. May be a Grafana question.

Reply:
It shouldn't be necessary to add anything to the file, but the regex expression you are using in your agent config may need to be tweaked. You can look at some of the recent integrations with logs in `cloud-onboarding` to see how the level is getting extracted.

Reply:
oh yeah haha. Updated the config